pytorch_lightning/setup_tools.py

```diff
@@ -55,107 +52,10 @@ def _load_requirements(path_dir: str , file_name: str = 'requirements.txt', comm
     return reqs
 
 
-def _parse_for_badge(
```

Comment:
Why did you delete those functions ?

Reply:
have not used them and the download was time-to-time failing